device-types/Extreme Networks/VSP-7254XSQ.yaml

```diff
@@ -0,0 +1,131 @@
+---
+manufacturer: Extreme Networks
+model: VSP-7254XSQ
+slug: extreme-networks-vsp-7254xsq
+part_number: VSP-7254XSQ
+u_height: 1
+is_full_depth: true
+weight: 7250
+weight_unit: g
+airflow: front-to-rear
+subdevice_role: parent
```

Comment:
This is generally used for chassis which house other devices, this is not meant for devices with modules. Is this actually needed, and if so why are there no `device-bays`?

Reply:
removed the parent role.